tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/01_filterStores.js

```diff
@@ -0,0 +1,212 @@
+require('module-alias/register');
+
+// Helpers to open and close browser
+const helper = require('@utils/helpers');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const contactPage = require('@pages/BO/shopParameters/contact');
+const storesPage = require('@pages/BO/shopParameters/stores');
+
+// Import data
+const {stores} = require('@data/demo/stores');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_contact_stores_filterStores';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+
```

Comment:
There are some empty lines that shouldn't be there :P

Reply:
all fixed
thanks